src/igvfd/schemas/matrix_file.json

```diff
@@ -132,22 +134,31 @@
                 "igvf_utils": "cell"
             }
         },
-        "dimension2": {
-            "title": "Dimension 2",
-            "description": "Second dimension of the matrix.",
-            "type": "string",
-            "enum": [
-                "cell",
-                "fragment",
-                "gene",
-                "time",
-                "treatment",
-                "variant",
-                "genomic position"
-            ],
+        "secondary_dimensions": {
+            "title": "Secondary Dimensions",
+            "description": "The secondary, tertiary....n levels of dimensions of the matrix.",
+            "type": "array",
+            "uniqueItems": true,
+            "minItems": 1,
+            "items": {
+                "title": "Secondary Dimensions",
```

Comment:
I think since this is the title of the items in the array, it can be singular here.
(The title of the overall property is still plural)